src/main/perl/ComponentProxy.pm

```diff
@@ -303,9 +278,10 @@ sub _load
     eval {
         local $SIG{__WARN__} = sub { push(@warns, $_[0]); };
         load $package;
+        $mod_fn = $self->_getComponentFilename();
```

Comment:
remove this from the eval, run this after the next if block, incl the `my $mod_fn = '';` from above